editor/project_manager.cpp

```diff
@@ -875,9 +876,16 @@ struct ProjectItem {
 		favorite = p_favorite;
 		grayed = p_grayed;
 	}
-	_FORCE_INLINE_ bool operator<(const ProjectItem &l) const { return last_modified > l.last_modified; }
+	_FORCE_INLINE_ bool operator<(const ProjectItem &l) const { 
+		if(orderedLatestModification)
+			return last_modified > l.last_modified; 
+		return project < l.project;
+	}
 	_FORCE_INLINE_ bool operator==(const ProjectItem &l) const { return project == l.project; }
+	public:
+	static void setOrderedLatestModification(bool value) { orderedLatestModification = value; }
```

Comment:
As you can see in surrounding code, method and variable identifiers in Godot's code base use `snake_case`, so you should do the same for new code.